bindings/c/test/apitester/fdb_c_api_tester.cpp

```diff
@@ -36,7 +36,7 @@ namespace FdbApiTester {
 
 namespace {
 
-#define API_VERSION_CLIENT_TMP_DIR 730
+#define API_VERSION_CLIENT_TMP_DIR 720
```

Comment:
Why did this get lowered?

Reply:
It reverts a recent change by Trevor. 720 is the version where CLIENT_TMP_DIR option was first introduced